sdk/nodejs/tests/runtime/props.spec.ts

```diff
@@ -60,6 +66,7 @@ class TestMocks implements runtime.Mocks {
                     id: runtime.isDryRun() ? undefined : "test-id",
                     state: {},
                 };
+            case "error":
```

Comment:
Nit: Can we throw a specific error for this case, rather than relying on the fall through for "unknown resource"?